lib/stack_master/commands/apply.rb

```diff
@@ -62,12 +62,16 @@ def create_or_update_stack
       end
 
       def create_stack
+        upload_files
+        options = stack_options.merge(tags: proposed_stack.aws_tags, change_set_type: 'CREATE')
+        @change_set = ChangeSet.create(options)
+        halt!(@change_set.status_reason) if @change_set.failed?
+        @change_set.display(StackMaster.stdout)
         unless ask?('Create stack (y/n)? ')
-          failed!("Stack creation aborted")
+          cf.delete_stack(stack_name: stack_name)
```

Comment:
Actually when you create a change set for a new stack, an empty stack in `REVIEW_IN_PROGRESS` status is also created.
Even though you delete the change set only, the empty stack still remains.
The stack can be deleted only by `delete_stack` API.
So I think SM should delete the stack when a user decides to not create a stack.